### DISCUSSION TITLE
[@types/send] uses the wrong types library for the mime package

### DISCUSSION THREAD

**Discussion by spaenleh** [DefinitelyTyped/DefinitelyTyped#70179] [Issues with a @types package]
It looks like the `@types/send` uses `@types/mime` when it should use the `@types/mime-types`.

From the source code in `send`:
https://github.com/pillarjs/send/blob/97741005edfb237c7f594aeb0a84e2a458b8e686/index.js#L23

And in the current version of @types/send the package depends on `@types/mime` instead of `@types/mime-types`.

I think there is an issue in the current typings, but please let me know if I got confused in some way.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
This is true for the newer versions of `send` (e.g. v1.2.0, the latest at the time of writing), but the 0.x versions still use `mime`: <https://github.com/pillarjs/send/blob/0.19.0/index.js#L24>

PR #73803 has been opened to add type definitions for v1.2.0 of send, which removes the dependency on `@types/mime`

**Reply by MysteryBlokHed** [reply]
PR merged :)
